controllers/cuenta_bancaria_banco.go

```diff
@@ -0,0 +1,76 @@
+package controllers
+
+import (
+	"strconv"
+
+	"github.com/astaxie/beego"
+	"github.com/udistrital/tesoreria_mid/helpers"
+	errorctrl "github.com/udistrital/utils_oas/errorctrl"
+)
+
+// CuentaBancariaBancoController operations for cuenta_bancaria_banco
+type CuentaBancariaBancoController struct {
+	beego.Controller
+}
+
+// URLMapping ...
+func (c *CuentaBancariaBancoController) URLMapping() {
+	c.Mapping("GetOne", c.GetOne)
+	c.Mapping("GetAll", c.GetAll)
+}
+
+// GetOne ...
+// @Title GetOne
+// @Description get Cuenta_bancaria_banco by id
+// @Param	id		path 	string	true		"The key for staticblock"
+// @Success 200 {object} models.Cuenta_bancaria_banco
+// @Failure 403 :id is empty
+// @router /:id [get]
+func (c *CuentaBancariaBancoController) GetOne() {
+	defer errorctrl.ErrorControlController(c.Controller, "CuentaBancariaBancoController")
+	// Validación entradas
+	idStr := c.Ctx.Input.Param(":id")
+	id, err := strconv.Atoi(idStr)
+	if err != nil {
+		panic(errorctrl.Error("GetOne", "Error en los parámetros de entrada id no entero", "400"))
+	}
+	// Lamada a helper
+	if cuentaBancaria, err := helpers.ObtenerCuentaBancariaBancoPorId(id); err == nil && cuentaBancaria != nil {
+		c.Data["json"] = map[string]interface{}{"Success": true, "Status": "200", "Message": "Request successful", "Data": cuentaBancaria}
+	} else {
+		panic(errorctrl.Error("GetOne", err, err["status"].(string)))
```

Comment:
Falta controlar en el `else` el escenario cuando `err == nil` Y `cuentaBancaria == nil`, podría ser algo así:
```suggestion
		if (cuentaBancaria == nil) {
			panic(errorctrl.Error("GetOne", "Cuenta no existe", "404"))
		}
		panic(errorctrl.Error("GetOne", err, err["status"].(string)))
```

Reply:
Se ajusta en commit https://github.com/udistrital/tesoreria_mid/commit/0411cccd93d5f8cf92ca51b6b302fae4ddb57e5e